ArduCopter/motor_test.cpp

```diff
@@ -149,6 +156,7 @@ uint8_t Copter::mavlink_motor_test_start(mavlink_channel_t chan, uint8_t motor_s
             g.failsafe_throttle = FS_THR_DISABLED;
             g.failsafe_battery_enabled = FS_BATT_DISABLED;
             g.failsafe_gcs = FS_GCS_DISABLED;
+            g.fs_ekf_action = 0;
```

Comment:
I'm 99% sure this doesn't prevent EKF failsafe (and that is why there is no macro for it). Setting `g.fs_ekf_thresh` will prevent it from enabling though.